src/data/sidenav.json

```diff
@@ -597,6 +597,10 @@
       }
     ]
   },
+  {
+    "displayName": "The Relicans",
```

Comment:
Though this is proposed to be a primary nav item, eventually it may be better suited under a section like Developer Relations, where we could put this content and our developer events. I'm ok with it being in the primary nav for now, but just something to consider.